README.md

```diff
@@ -1,38 +1,29 @@
 # Spreed.ME Nextcloud app
-- This app can only be used in conjunction with a [Spreed WebRTC server](https://github.com/strukturag/spreed-webrtc).
-  You will not be able to use this app without such a server, but it's **easy to set up** one – The whole process only takes about 5-10 minutes – 2 minutes if you are fast ;).
-- The latest version of this app can be found in the Nextcloud app store or at [github.com/strukturag/nextcloud-spreedme](https://github.com/strukturag/nextcloud-spreedme)
+
+- This app can only be used together with a [Spreed WebRTC server](https://github.com/strukturag/spreed-webrtc) which is available as open source software (AGPL license) just like Nextcloud.
+- This app requires HTTPS.
+- The latest version of this app can be found in the Nextcloud app store or at [github.com/strukturag/nextcloud-spreedme](https://github.com/strukturag/nextcloud-spreedme).
 
 ## Features
+
 - Securely communicate with your friends and family using rich audio-, video- and text chat right from your Nextcloud installation – in your browser
 - Share presentations and other documents and save them to your Nextcloud
 - Share your Nextcloud presentations and other documents
 - Share your computer screen
 - Share YouTube videos
-- Much, much more :)
-
-## Preparations
-Before setting up this app (+ Spreed WebRTC) you need to ask yourself a few questions:
-
-1. Do you want to run Nextcloud and Spreed WebRTC on the same origin (i.e. on the same domain+port combination)?
-   (e.g. access Nextcloud at https://mynextcloudserver.com:8443 and Spreed WebRTC at https://mynextcloudserver.com:8443/webrtc/) **-->** Then use **/webrtc/** as your **basePath** for the next steps.
-2. Do you want to run Nextcloud and Spreed WebRTC on two different origins?
-   (e.g. access Nextcloud at https://mynextcloudserver.com:8443 and Spreed WebRTC at https://webrtc.mynextcloudserver.com:8080) **-->** Then use an empty / no **basePath** for the next steps.
-
-**Note:** We recommend using option **1.** as it's much easier to maintain. Also it seems that you can't use Screensharing in latest browsers when using option **2.**. We're working on a workaround so Screensharing also works with option **2.**, but for now option **1.** seems the way to go.
+- ...
 
 ## Installation / Setup of this app
-First off, here's a little encouragement:
-This list of steps might seem a bit long, but it's really easy to follow – we promise. You only have to set up this app once :)
+
+This set of installation steps assume that you already have set up Nextcloud using a web server like Apache or Nginx and will help you to set up Spreed WebRTC below the `/webrtc` subpath at your existing web server. If your Nextcloud server is not using SSL yet, you need to [enable SSL now](https://docs.nextcloud.com/server/9/admin_manual/installation/source_installation.html#enabling-ssl-label), as Spreed WebRTC does require an encrypted connection.
```

Comment:
Technically speaking the NC instance also has to run over https, as pages accessing `getUserMedia` via an Iframe have to use a secure connection, too.